<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePassowrd</class>
 <widget class="QDialog" name="ChangePassowrd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1167</width>
    <height>824</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">border-image: url(:/new/prefix1/ATM/New folder/New folder/istockphoto-1170700236-612x612.jpg);</string>
  </property>
  <widget class="QLineEdit" name="NewPass">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>330</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1/ATM/ATM_data.xlsx);
font: 900 18pt &quot;Segoe UI Black&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>300</y>
     <width>531</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1/ATM/ATM_data.xlsx);
color: rgb(0, 0, 0);
font: 700 30pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>New Password                   :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="OldPass">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>240</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1/ATM/ATM_data.xlsx);
font: 900 18pt &quot;Segoe UI Black&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>210</y>
     <width>531</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1/ATM/ATM_data.xlsx);
color: rgb(0, 0, 0);
font: 700 30pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Old Passowrd                     :</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="NewPassCom">
   <property name="geometry">
    <rect>
     <x>740</x>
     <y>430</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1/ATM/ATM_data.xlsx);
font: 900 18pt &quot;Segoe UI Black&quot;;
</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>400</y>
     <width>531</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1/ATM/ATM_data.xlsx);
color: rgb(0, 0, 0);
font: 700 30pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Confermation Password   :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>610</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1/ATM/ATM_data.xlsx);
color: rgb(0, 0, 0);
font: 700 30pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>&lt;Go back </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>660</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1/ATM/ATM_data.xlsx);
color: rgb(0, 0, 0);
font: 700 30pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Cancel </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>710</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/1/ATM/ATM_data.xlsx);
color: rgb(0, 0, 0);
font: 700 30pt &quot;Segoe UI&quot;;
</string>
   </property>
   <property name="text">
    <string>Next &gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
